docs/Release-Notes-2.16.md

```diff
@@ -0,0 +1,123 @@
+
+```
+$ mock -r epel-8-x86_64 --shell
+ERROR: Could not find required config file: /etc/mock/epel-8-x86_64.cfg
+ERROR: There are those alternatives:
+ERROR: 
+ERROR: [1] alma+epel-8-x86_64
+ERROR:     Use instead: mock -r alma+epel-8-x86_64 --shell 
+ERROR:     Builds against AlmaLinux 8 repositories, together with the official EPEL repositories.
+ERROR:     Project page: https://almalinux.org/
+ERROR:     Enable permanently by:
+ERROR:     $ ln -s /etc/mock/alma+epel-8-x86_64.cfg /home/praiskup/.config/mock/epel-8-x86_64.cfg
+ERROR: 
+ERROR: [2] centos+epel-8-x86_64
+ERROR:     Use instead: mock -r centos+epel-8-x86_64 --shell 
+ERROR:     Builds against CentOS Linux 8 repositories, together with the official EPEL repositories.
+ERROR:     This will reach end-of-life in January 2021.
+ERROR:     Enable permanently by:
+ERROR:     $ ln -s /etc/mock/centos+epel-8-x86_64.cfg /home/praiskup/.config/mock/epel-8-x86_64.cfg
+ERROR: 
+ERROR: [3] rhel+epel-8-x86_64
+ERROR:     Use instead: mock -r rhel+epel-8-x86_64 --shell 
+ERROR:     Builds against Red Hat Enterprise Linux 8 repositories, together with the official EPEL repositories.
+ERROR:     This mimics what is done in the official EPEL build system, but you need a Red Hat subscription:
+ERROR:     https://rpm-software-management.github.io/mock/Feature-rhelchroots
+ERROR:     Enable permanently by:
+ERROR:     $ ln -s /etc/mock/rhel+epel-8-x86_64.cfg /home/praiskup/.config/mock/epel-8-x86_64.cfg
+ERROR: 
+ERROR: [4] rocky+epel-8-x86_64
+ERROR:     Use instead: mock -r rocky+epel-8-x86_64 --shell 
+ERROR:     Builds against Rocky Linux 8 repositories, together with the official EPEL repositories.
+ERROR:     Project page: https://rockylinux.org/
+ERROR:     Enable permanently by:
+ERROR:     $ ln -s /etc/mock/rocky+epel-8-x86_64.cfg /home/praiskup/.config/mock/epel-8-x86_64.cfg
+```
+
+Additional changes are:
+
+ * Fedora 33 configs were moved to eol/ directory
+ * EOLed EPEL Playground configs
+ * Added configs for CentOS Stream 9 + EPEL Next 9
+ * Added configs for epel8-playground
+ * We expanded `dnf_vars` which cause an issue on EL7 hosts [RHBZ#2026571](https://bugzilla.redhat.com/show_bug.cgi?id=2026571)
+ * Added compatibility symlinks for EPEL 7 to centos+epel-7-*
+ * Resolved the multiple "local" repo problems
+ * Dropped rhel+epel-8-ppc64 config
+ * Added rhel+epel-8-s390x config
+ * Added navy-8-x86_64 config
+ * Reduced packages installed in epel chroots
+
+## Mock mock 2.16 changes:
+
+Mock got a new configuration option:
+
+```
+config_opts["no-config"]["epel-8"] = {}
+config_opts["no-config"]["epel-8"]["alternatives"] = {
+    "alma+epel-8": {
+        "description": [
+            "Builds against AlmaLinux 8 repositories, "
+            "together with the official EPEL repositories.",
+            "Project page: https://almalinux.org/"
+        ],
+    }
+    }
+```
```

Comment:
I for one don't think this should be documented, it is mostly internal/implementation detail.  But I don't object ...

Reply:
It **can** be used by users. And the release notes will be actually the only place where it is documented. :) I think it is worth mentioning here.